javascript/builtins/Map.json

```diff
@@ -473,6 +473,74 @@
             }
           }
         },
+        "groupBy": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Map/groupBy",
+            "spec_url": "https://tc39.es/proposal-array-grouping/#sec-map.groupby",
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": [
+                {
+                  "version_added": "preview",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "javascript.options.experimental.array_grouping",
+                      "value_to_set": "true"
+                    }
+                  ]
+                },
+                {
+                  "version_added": "98",
+                  "version_removed": "preview",
+                  "flags": [
+                    {
+                      "type": "preference",
+                      "name": "javascript.options.experimental.array_grouping",
+                      "value_to_set": "true"
+                    }
+                  ],
+                  "alternative_name": "Array.prototype.groupToMap"
+                }
+              ],
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": [
+                {
+                  "version_added": "preview"
+                },
+                {
+                  "version_added": "16.4",
+                  "version_removed": "preview",
```

Comment:
I "think" thisneeds to be a specific version.

Reply:
It's not released yet. The browsers I have access to are 16.5 and TP and it's in TP, but I don't know if it's in 17.0.

Reply:
Ah, thanks. I don't know what BCD does for this case. Maybe exactly this.